tests/test_dataframe.py

```diff
@@ -145,30 +157,36 @@ def test_dataframe_display_repr_html(db: gp.Database):
     assert (t.order_by("id")[:]._repr_html_()) == expected
 
 
-def test_dataframe_display_repr_empty_result(db: gp.Database):
+def test_dataframe_display_repr_zero_rows(db: gp.Database):
     # fmt: off
     rows = [(1, "Lion",), (2, "Tiger",), (3, "Wolf",), (4, "Fox")]
     # fmt: on
     t = db.create_dataframe(rows=rows, column_names=["id", "animal"])
-    assert str(t[lambda t: t["id"] == 0]) == "  \n--\n(0 rows)\n"
-    assert (t[lambda t: t["id"] == 0]._repr_html_()) == ""
+    t = t[lambda t: t["id"] == 0]
+    print(t)
+    assert str(t) == "----\n" "----\n" "----\n" "(0 rows)\n"
+    assert (t._repr_html_()) == ""
 
 
 def test_dataframe_display_result_null(db: gp.Database):
     # fmt: off
     rows = [([1,1,1], None,), ([2,2,2], "Tiger",), ([3,3,3], None,), ([4,None,4], "Fox")]
     # fmt: on
     t = db.create_dataframe(rows=rows, column_names=["id", "animal"])
+    t = t.order_by("id")[:]
+    print(t)
     expected = (
+        "-----------------------\n"
         " id           | animal \n"
-        " ------------ + ------ \n"
+        "--------------+--------\n"
         " [1, 1, 1]    |        \n"
         " [2, 2, 2]    | Tiger  \n"
         " [3, 3, 3]    |        \n"
         " [4, None, 4] | Fox    \n"
+        "-----------------------\n"
         "(4 rows)\n"
```

Comment:
Can we add a test to show `(0 rows)`? I failed since we cannot do that with `from_rows()`. (the empty `VALUES` problem).